src/components/addHouse.jsx

```diff
@@ -0,0 +1,114 @@
+import React, { useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { useNavigate } from 'react-router-dom';
+import { sendHouses } from '../redux/houses/housesSlice';
+
+const HouseForm = () => {
```

Comment:
- [ ] When I try to add a house, I get an internal server error:

![image](https://github.com/mercymugambi/frontend_house_reservations_react_app/assets/57421823/c064abd7-56eb-4034-83e8-50396535c7cc)

Kindly, fix this bug so your app works as intended. 

_note: This means I couldn't test the reservations functionality.  This will be taken a look in the next review._